src/components/LoadingModal.tsx

```diff
@@ -7,12 +8,14 @@ interface ILoadingModalProps {
 }
 
 const LoadingModal = ({ isVisible = true, children }: ILoadingModalProps) => {
+  const [isTxnProcessing] = useMCStore((s) => [s.isTxnProcessing]);
```

Comment:
Hi @GHesericsu , just wondering if this is for hiding the modal when user rejects a sign?